### ISSUE TITLE
[Android] Game still crashes when I choose certain dialogue options with Tandi

### ISSUE THREAD

**Issue by Vonklinkenhoffn** [alexbatalov/fallout1-ce#110]
Hi
First let me just say how awesome I find it to play Fallout on my mobile, thank you for this!

I have found a rather annoying bug though. 
Everytime I ask Tandi either "What else is around theses parts?" or "Have you heard any stories from down south?" the game crashes. 
I've read through some posts here and found  a post about this very issue that says it was supposed to be fixed sometime in february. I could've misunderstood, but anyway the bug is still happening for me though.

I have a Samsung Galaxy S8 btw.

**Comment by Vonklinkenhoffn** [comment]
@alexbatalov I've found a post that says "use latest [debug](https://github.com/alexbatalov/fallout1-ce/actions) version, I'll prepare new release with hot fixes later this week."

However I don't know how to use latest debug version on Android. I've looked through the files, but I'm way too uncertain how to go from installing a simple apk-file to copy over the files from the debug version. I see there's some files that seem to be for iOS in there and such. I may have misunderstood something though. 

Also I figured that if the hotfixes was in, as you said in that post in february, then it shouldn't still be crashing when talking to Tandi, but unfortunately it still does. 

Sorry for the long post, I just figured I'd show that I've really tried fixing this on my own without bothering you. I still failed though lol.

**Comment by alexbatalov** [comment]
Let me double check this, please post your save game.

**Comment by Vonklinkenhoffn** [comment]
Well of course, certainly. Here you go, and thank you for looking into this and responding so quickly.

[SLOT06.zip](https://github.com/alexbatalov/fallout1-ce/files/12838777/SLOT06.zip)

**Comment by alperkose** [comment]
Looks like it's the same issue described in #76 The issue is actually fixed but there is no release containing the fix, yet. I also had the same issue which I get it fixed by rebuilding the executable. You can find my response there for options on rebuilding the executable.

**Comment by Vonklinkenhoffn** [comment]
Yes, I found that one too and wondered if it might be the same. I don't have the means to rebuild the executable myself though, I'm on a trip and only have my phone with me. Sometimes I can borrow my wifes laptop when I have the time though, but it's barely enough to just extract the save-files from my phone or do some small cfg stuff. 
I did manage to play on just not asking all the questions from Tandi though.

However, what's really ruining the game for me now is rather the other issue I reported on yesterday, when it crashes whenever I try to enter vault 15. That one is really a game-stopper. I don't know how to fix this. 
Isn't there just a simple apk file with a beta or test build somewhere I could install on my phone maybe? 
That would've been great!

Is there any new release on the horizon too btw?
